src/coprocessor/dag/expr/ctx.rs

```diff
@@ -45,8 +45,24 @@ pub const FLAG_OVERFLOW_AS_WARNING: u64 = 1 << 6;
 // FLAG_DIVIDED_BY_ZERO_AS_WARNING indicates if DividedByZero should be returned as warning.
 pub const FLAG_DIVIDED_BY_ZERO_AS_WARNING: u64 = 1 << 8;
 
-pub const MODE_NO_ZERO_DATE_MODE: u64 = 25;
-pub const MODE_ERROR_FOR_DIVISION_BY_ZERO: u64 = 27;
+bitflags! {
+    /// Please refer to SQLMode in `mysql/const.go` in repo `pingcap/parser` for details.
+    pub struct SqlMode: u64 {
+        const STRICT_TRANS_TABLES = 1 << 22;
+        const STRICT_ALL_TABLES = 1 << 23;
+        const NO_ZERO_IN_DATE = 1 << 24;
+        const NO_ZERO_DATE = 1 << 25;
+        const INVALID_DATES = 1 << 26;
+        const ERROR_FOR_DIVISION_BY_ZERO = 1 << 27;
+    }
+}
+
+impl SqlMode {
+    /// Returns if 'STRICT_TRANS_TABLES' or 'STRICT_ALL_TABLES' mode is set.
```

Comment:
Nit: this comment is not useful. It would be better to describe what 'strict' mode means, the effect it will have on execution, etc. (or you could just remove it)

Reply:
Good idea, thanks!